packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -78,32 +125,85 @@ const styles = StyleSheet.create({
         top: 8,
     },
 
-    column: {
-        boxSizing: "border-box",
-        flex: "0 0 50%",
+    containerNoSidebar: {
+        minHeight: "auto",
```

Comment:
What does this rule do?

Reply:
That was overriding the minHeight defined in `container`, but I think I'm going to try and reformulate this so it isn't necessary.